### ISSUE TITLE
Missing functions in version 1.0.3

### ISSUE THREAD

**Issue by FerandDalatieh** [jimhester/lintr#357]
I used to use lintr_1.0.2.9000 from its github repository, but now I checked out the new released version on CRAN lintr_1.0.3 and I am missing some useful functions like the followings:
```
todo_comment_linter
object_name_linter
function_left_parentheses_linter
T_and_F_symbol_linter
semicolon_terminator_linter
pipe_continuation_linter
```
Is there any new functions which can be used as a replacement of them?

Thanks!

**Comment by jimhester** [comment]
I bumped the versions of the devel repo and the news to make this more clear https://github.com/jimhester/lintr/commit/707bf03f1ee0193f38c9f357b15f197cf0984452

**Comment by Fuco1** [comment]
I don't understand your comments @jimhester. I've installed `lintr` with `install.packages` and I got version 1.0.3 and the above functions don't seem to work, eventhough when I look at the 1.0.3 tag in this repo and search for them they are present.

This is the setting I'm using: `"with_defaults(default = default_linters, object_name_linter(\"lowerCamelCase\"))"` and I get no linting at all.  When I remove then `object_name_linter` I get output.
